app/routers/monitoring.py

```
@@ -0,0 +1,451 @@
+    async def send_personal_message(self, message: str, websocket: WebSocket):
+        try:
+            await websocket.send_text(message)
+        except:
+            self.disconnect(websocket)
+    
+    async def broadcast(self, message: str):
+        disconnected = []
+        for connection in self.active_connections:
+            try:
+                await connection.send_text(message)
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)